contents/careers/marketer.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Marketer
+sidebar: Careers
+showTitle: true
+---
+## PostHog exists to increase the number of successful products in the world.
+
+We're growing quickly and are looking for our first marketer to…… While we have mainly engineers to date, we are now hiring across all areas of the business.
+We also would ideally love to bring on someone who is interested in xxxx.
+
+## What you’ll be doing
+*
+*
+*
+
+
+## We hire the best to tell us what to do
+
+We are focused on increasing user acquisition, retention and our community growth. Marketing is core to all of these things.
+
+Success in this role means xxx
```

Comment:
* The primarily goal is that we increase the number of new users. We currently get around 120 deployments a week organically, but we want to accelerate this growth.
* Secondarily, we increase our community size (it currently exists in our Slack user group, GitHub, our mailing list and Twitter)

We'd expect to give you a healthy budget to achieve the above, a great candidate will be able to steer us as to what type of spend is appropriate.